### PR TITLE
Add icons

### PR DESCRIPTION
**Pull request by lanker**
Also update favicon and icon in manifest

### CODE REVIEW

**Comment by thepatrik** [comment]
Totally awesome!

File: public/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/pixlcrypt_mono.png">
```
Comment (thepatrik) [inline public/index.html]: Specify image type for png? https://stackoverflow.com/questions/6121725/favicon-ico-or-png-correct-tags

File: public/manifest.json [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
-  "name": "Create React App Sample",
+  "short_name": "pixlcrypt",
+  "name": "pixlcrypt",
   "icons": [
     {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
+      "src": "pixlcrypt_mono.png",
+      "sizes": "128.128"
```
Comment (thepatrik) [inline public/manifest.json:7]: type?

Comment (lanker) [reply public/manifest.json:7]: not needed

File: public/index.html (hunk 1 above)
Comment (lanker) [reply public/index.html]: ok!

**Review by thepatrik — APPROVED** [review]

**Pull request merged**